src/Pos/Util/JsonLog.hs

```diff
@@ -177,3 +177,6 @@ jsonLogDefault x = do
     JL.jsonLogDefault jlc x
 
 deriving instance CanJsonLog (t m) => CanJsonLog (TaggedTrans tag t m)
+
+-- Required for @Explorer@ @BListener@ redirect
+deriving instance CanJsonLog m => CanJsonLog (TaggedTrans tag IdentityT m)
```

Comment:
Yeah, seems the previous more general instance doesn't quite work. I added the additional one since we need that one for _explorer_. I guess I could remove the previous instance, and yes, I don't know why it doesn't work.